cmd/ingress-perf.go

```diff
@@ -31,6 +32,14 @@ var cmd = &cobra.Command{
 	Short: "Benchmark OCP ingress stack",
 }
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "😎 Print the version number of ingress-perf",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("%s@%s \n", version.Version, version.GitCommit)
```

Comment:
This would print as requested in the RFE
```
./bin/ingress-perf version
version-branch@0a412d04ae6690d95eec2b5aa740c794ae08b8f6 
```
or would you like this to be printed like in the kube-burner?
```
kube-burner version
Version: 1.5
Git Commit: db331979da3447c210ce308a4c1d5e473ba7d440
Build Date: 2023-04-05T17:02:00Z
Go Version: go1.19.7
OS/Arch: darwin arm64
```

Reply:
@rsevilla87 can you ptal?